src/SurveyService/Controllers/GroupController.cs

```diff
@@ -0,0 +1,21 @@
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.SurveyService.Business.Commands.Group.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Group;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.SurveyService.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class GroupController : ControllerBase
+{
+  [HttpPost("create")]
+  public async Task<OperationResultResponse<Guid>> CreateAsync(
+    ICreateGroupCommand command,
```

Comment:
May be add attribute [FromServices]?

Reply:
Trying to be consistent with other code, DI will work without that.

Reply:
In other controllers you use the attribute, but DI would have worked without that?
@vorobevamv

Reply:
I'm not sure. Here can be a lot of sources (services, body, query). Even if it works, it is difficult to read code.
You can try to run without attributes, then tell us))